int8 num_colors
bool show_chart
---
std_msgs/Int16MultiArray color_mask
